app/src/main/java/com/example/bigowlapp/model/UserScheduleResponse.java

```diff
@@ -0,0 +1,43 @@
+package com.example.bigowlapp.model;
+
+import com.google.firebase.Timestamp;
+
+public class UserScheduleResponse {
+    private Response response;
+    private Timestamp responseTime;
+    private boolean exists;
```

Comment:
The exists variable should be removed since it's a useless variable. The issue with firebase has been fixed.